app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,369 @@
+            android:id="@+id/storyline_tv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_x4"
+            android:layout_marginTop="@dimen/margin_x6"
+            android:text="@string/storyline"
+            android:textColor="@color/white"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/reviews_tv" />
+
+        <TextView
+            android:id="@+id/plot_tv"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_x4"
+            android:layout_marginEnd="@dimen/margin_x4"
+            android:layout_marginTop="@dimen/margin_x1"
+            android:lineHeight="20dp"
+            android:text="@string/plot_text"
+            android:textColor="@color/menu_text_color"
+            android:textSize="14sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/storyline_tv" />
+
+        <TextView
+            android:id="@+id/cast_tv"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_x4"
+            android:layout_marginTop="@dimen/margin_x6"
+            android:layout_marginEnd="@dimen/margin_x4"
+            android:text="@string/cast"
+            android:textColor="@color/white"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/plot_tv" />
+
+        <ImageView
+            android:id="@+id/downey_jr_iv"
+            android:scaleType="fitCenter"
+            android:adjustViewBounds="true"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x2"
+            android:contentDescription="@string/robert_downey_jr"
+            android:src="@drawable/downey_jr"
+            app:layout_constraintEnd_toStartOf="@+id/evans_iv"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/cast_tv" />
+
+        <TextView
+            android:id="@+id/downey_jr_tv"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x1"
+            android:text="@string/robert_downey_jr"
+            android:textColor="@color/white"
+            android:textSize="@dimen/common_text_size"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/downey_barrier_start"
```

Comment:
What is the idea for barrier? Can we just connect to the start and end of the image view?

Reply:
The barrier was used to align textview to the imageview borders in order to fill the width of the imageview. I thought that another approach with connecting the start and the end is not going to work, but it appears to be working.